sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -363,7 +363,7 @@ public static void notifyUserConfirmation(@UserConfirmationDef int userConfirmat
 
             @Override
             public void run() {
-                future.complete(ErrorLogHelper.getNewMinidumpDirectory().getAbsolutePath());
+                future.complete(ErrorLogHelper.getNewMinidumpSubfolderWithDeviceInfo(mContext).getAbsolutePath());
```

Comment:
What if we add other files like session/userId later? I think the name could be simplified like WithContextData or something like that.